src/query/vacuum.c

```diff
@@ -2847,9 +2847,9 @@ vacuum_process_vacuum_data (THREAD_ENTRY * thread_p)
   vacuum_finalize (thread_p);
 
   vacuum_Data.is_vacuum_complete = true;
-
-  PERF_UTIME_TRACKER_TIME (thread_p, &perf_tracker, mnt_vac_master_time);
 #endif /* SA_MODE */
+
+  PERF_UTIME_TRACKER_TIME (thread_p, &perf_tracker, PSTAT_VAC_MASTER);
```

Comment:
@asturzu @ribram 

I'm not sure this should also be applied to `SERVER_MODE`. I think the existing code is correct. 
Would you confirm it again?